github/node/express/kubernetesCosmosDB/ArmTemplates/aks-applicationInsights.json

```diff
@@ -43,15 +43,5 @@
                 "name": "[parameters('databaseAccountId')]"
             }
         }
-    ],
-    "outputs": {
```

Comment:
may I know the reason why are you removing this as it is there in Azure pipelines arm templates?

Reply:
@anuragc617 This is done to workaround this [issue](https://github.com/Azure/arm-deploy/issues/27)